### ISSUE TITLE
Have only ["loss"] in keras_model.metrics_names

### ISSUE THREAD

**Issue by AndreyStille** [matterport/Mask_RCNN#2480]
Please help
Problem:
`model.keras_model.metrics_names `
retruns just ['loss']
Expectation:
[ "rpn_class_loss",  "rpn_bbox_loss", "mrcnn_class_loss", "mrcnn_bbox_loss", "mrcnn_mask_loss"]
As in model.py  we have:
```
for name in loss_names:
            if name in self.keras_model.metrics_names:
                continue
            layer = self.keras_model.get_layer(name)
            self.keras_model.metrics_names.append(name)
            loss = (
                tf.reduce_mean(layer.output, keepdims=True)
                * self.config.LOSS_WEIGHTS.get(name, 1.))
            self.keras_model.metrics.append(loss)
```
But after compiling and durin training I recieve only one loss
![12344](https://user-images.githubusercontent.com/65295663/107730122-64488f00-6cfb-11eb-9c5d-3db6616047a1.png)
